lib/perl/Genome/SoftwareResult/User.pm

```diff
@@ -140,28 +127,11 @@ sub _register_users {
     }
 }
 
-sub _lock_and_create_if_needed {
+sub get_or_create {
     my $class = shift;
     my $params = shift;
 
-    my $existing = $class->get(%$params);
-    return if $existing;
-
-    my $resource = $class->_resolve_lock_name($params);
-    my $lock = Genome::Sys::LockProxy->new(
-        resource => $resource,
-        scope => 'site',
-    )->lock();
-    $class->_load_or_create($params);
-
-    return $lock;
-}
-
-sub _load_or_create {
-    my $class = shift;
-    my $params = shift;
-
-    my $self = $class->load(%$params);
+    my $self = $class->get(%$params);
```

Comment:
Nope, it sure doesn't.  It makes me think it would be nice to eventually unify `Genome::Utility::ObjectWithLockedConstruction` and `Genome::SoftwareResult`.  I'll just remove `get_or_create` entirely.